firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -444,7 +444,7 @@ private String readExistingIidOrCreateFid(PersistedInstallationEntry prefs) {
 
   /** Registers the created Fid with FIS servers and update the persisted state. */
   private PersistedInstallationEntry registerFidWithServer(PersistedInstallationEntry prefs)
-      throws IOException {
+      throws FirebaseInstallationsException, IOException {
```

Comment:
Can we just throw 1 exception type here?
Or where happens the unification (of exception types) for users?